tests/Functional/Fixtures/symfony/config/bundles.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
+    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
+    \Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
+    \Joli\SeoOverride\Bridge\Symfony\SeoOverrideBundle::class => ['all' => true],
```

Comment:
```suggestion
    Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
    Joli\SeoOverride\Bridge\Symfony\SeoOverrideBundle::class => ['all' => true],
```

It much more beautiful 🎨 💅🏼